account_statement_import_camt/models/account_statement_import.py

```diff
@@ -30,8 +30,8 @@ def _parse_file(self, data_file):
                         )
                         transactions.extend(new)
                 return currency, account_number, transactions
-            except (zipfile.BadZipFile, ValueError):
-                pass
+            except (zipfile.BadZipFile, ValueError) as err:
+                _logger.info(str(err))
```

Comment:
Please don't log the error. this is simply a test to see if the file can be unzipped.

Reply:
it's to fix pylint

Reply:
maintainers complain if I mute such one

Reply:
Is it the maintainers that complain or pylint?

Reply:
Or do you mean, the (human) maintainers complained earlier on when you applied an instruction for pylint to ignore the muted exception? I'm pretty sure that muting is the correct approach here.

Reply:
Sorry for the misunderstanding.
1. `pylint` throws an error on just `pass`
2. As in most cases I got a request from reviewers not to use `pylint: disable=W<number>` I fixed here with `logger`

Reply:
I'll update this PR with pylint ignore if you're sure.
Sorry, if I wasn't polite or something like that. As many reviewers as many opinions.

Reply:
No, it was just unclear. I think whether or not to apply the `pylint: disable=` comment depends very much on the situation. Please apply it here.

Reply:
For the record, when I'm talking about 'forgiveness vs. permission' I'm talking about the code structure, not about our interaction ;-) See https://docs.quantifiedcode.com/python-anti-patterns/readability/asking_for_permission_instead_of_forgiveness_when_working_with_files.html. In such cases, it is very common to `pass` an exception.

Reply:
thanks for explanation, it's really helpful

Reply:
is it ok now for your approval?

Reply:
I'm suggesting to remove this particular check here: https://github.com/OCA/pylint-odoo/issues/361